src/Microsoft.ML.CpuMath/CpuMathUtils.netstandard.cs

```diff
@@ -19,11 +19,17 @@ internal static partial class CpuMathUtils
         public static int GetVectorAlignment()
             => Vector128Alignment;
 
+        /// <summary>
+        /// Check if <paramref name="a"/>'s alignment is suitable to SSE instructions. Returns <see langword="true"/>
+        /// if <paramref name="a"/>'s alignment is ok and <see langword="false"/> otherwise.
+        /// </summary>
+        /// <param name="a">The vector being checked.</param>
+        /// <returns>Whether <see langword="true"/> is aligned well.</returns>
```

Comment:
><see langword="true"/> [](start = 29, length = 22)

<paramref name="a"/>?

Reply:
<paramref name="a"/>

Reply:
"<paramref name="a"/>" ?

Reply:
You're right. Thanks.

---
In reply to: [273743672](https://github.com/dotnet/machinelearning/pull/3124#discussion_r273743672) [](ancestors = 273743672)